### DISCUSSION TITLE
Collaborative Databases for Non-Profit Coordination?

### DISCUSSION THREAD

**Discussion by quinn-p-mchugh** [life-itself/community#892] [General]
Hi all, I'm new to Life Itself.

I see that this community is interested in problems related to collective action and cooperation and another member suggested I post here.

**I'm wondering if there's a community specifically focused on building tools that allow *non-technical* people to build collaborative databases *with revision control (not just history)*?**

I think the key here is a tool that:
1. represents data in a tabular format
2. is easy for non-technical stakeholders to use
3. has revision control (not just history)
4. is able to accommodate relational data. 

Basically, a relational database with a [Gitbook](https://www.gitbook.com/)-like front-end where people can easily update some data and submit a change request for review. Local data storage would be icing on the cake.

Maybe something like [DokuWiki]( https://www.dokuwiki.org/dokuwiki) is a fine solution (just use the built-in tables), but this I still doesn't address the relational aspect. I'm basically imagining a tool that combines the best parts of [Dolt](https://www.dolthub.com/) and Gitbook and am finding it really hard to believe that a solution hasn't emerged for this problem.

**Context:** there are numerous non-profits in farmed animal advocacy (and almost certainly other social causes) that have similar goals and needs and also directly benefit from coordinating with one another to achieved outsized impact. 

In order to properly coordinate, however, these non-profits typically need some kind of tabular source of truth to track, for example, the status of campaigns being worked on. In farmed animal advocacy, Google Sheets is often used for this purpose, which introduces significant challenges related to data quality and labor costs associated with maintaining these spreadsheets.

**Are there any folks who have solved this specific problem? If not, who might I want to talk to?**

**Comment by rufuspollock** [comment]
Thanks for posting. I've been involved in building tools a bit like this in various areas for over 15y.

My instinct here would be to build a simple editor wrapper around git(hub) + CSV or git(hub) + markdown - see #829. Even better this wrapper would abstract away git for almost all users (i.e. it would auto-generate a pull request in the background for the user etc etc).

Right now, i've been building quite a few workflows trying to use obsidian.md as the editor and  markdown with frontmatter as the storage structure.

We've also been involved in building a gitbook like frontend called Flowershow over the last few years - https://flowershow.app

I can sketch more detail on this - and welcome questions to prompt further details.

**Reply by rufuspollock** [reply]
@quinn-p-mchugh wanted to check you had seen this ...
